data/map planets.txt

```diff
@@ -2877,7 +2877,7 @@ planet Muspel
 		fleet "Small Deep Security" 14
 
 planet Mutiny
-	attributes pirate
+	attributes core pirate "core pirate"
```

Comment:
```suggestion
	attributes core "core pirate" pirate
```
Alphabetically :)

Reply:
Damn, I can't read. Thanks for catching this!